src/Farmer/Arm/SiteExtensions.fs

```diff
@@ -0,0 +1,41 @@
+[<AutoOpen>]
```

Comment:
This file should be moved into `Web.fs`.

Reply:
I'll do that but I'd like to know the reasoning.

Reply:
The pattern we have is that all ARM resources are organised into a module that they live under e.g. Web, StorageAccount, Compute etc.